src/main/java/competition/electrical_contract/CompetitionContract.java

```diff
@@ -153,4 +153,7 @@ public DeviceInfo getUpperArmEncoder() {
     public DeviceInfo getLowerArmBrakeSolenoid() {
         return new DeviceInfo(1, false);
     }
+
+    public DeviceInfo getCollectorMotor(){ return new DeviceInfo(0,false);}
```

Comment:
Make sure to get "correct" values for this from the Electrical Contract document or directly from the electrical team (who should add them to the document)